src/core_plugins/metrics/public/components/yes_no.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react';
+
+function YesNo(props) {
```

Comment:
so some of the components are functions and some use the class syntax ... should we unify this ? or is there a reason behind this ?